src/libs/Middleware/Logging.ts

```diff
@@ -64,18 +65,20 @@ const Logging: Middleware = (response, request) => {
                 // incorrect url, bad cors headers returned by the server, DNS lookup failure etc.
                 Log.hmmm('[Network] API request error: Failed to fetch', logParams);
             } else if (
-                [
-                    CONST.ERROR.IOS_NETWORK_CONNECTION_LOST,
-                    CONST.ERROR.NETWORK_REQUEST_FAILED,
-                    CONST.ERROR.IOS_NETWORK_CONNECTION_LOST_RUSSIAN,
-                    CONST.ERROR.IOS_NETWORK_CONNECTION_LOST_SWEDISH,
-                    CONST.ERROR.IOS_NETWORK_CONNECTION_LOST_SPANISH,
-                ].includes(error.message)
+                (
+                    [
+                        CONST.ERROR.IOS_NETWORK_CONNECTION_LOST,
+                        CONST.ERROR.NETWORK_REQUEST_FAILED,
+                        CONST.ERROR.IOS_NETWORK_CONNECTION_LOST_RUSSIAN,
+                        CONST.ERROR.IOS_NETWORK_CONNECTION_LOST_SWEDISH,
+                        CONST.ERROR.IOS_NETWORK_CONNECTION_LOST_SPANISH,
+                    ] as string[]
+                ).includes(error.message)
```

Comment:
Instead of asserting you can use .some here 😄 
```suggestion
[
                    CONST.ERROR.IOS_NETWORK_CONNECTION_LOST,
                    CONST.ERROR.NETWORK_REQUEST_FAILED,
                    CONST.ERROR.IOS_NETWORK_CONNECTION_LOST_RUSSIAN,
                    CONST.ERROR.IOS_NETWORK_CONNECTION_LOST_SWEDISH,
                    CONST.ERROR.IOS_NETWORK_CONNECTION_LOST_SPANISH,
                ].some((message) => message === error.message)
```